src/cpacs_other/CTiglUIDManager.h

```diff
@@ -113,6 +113,11 @@ class CTiglUIDManager
     // Clears the uid store
     TIGL_EXPORT void Clear();
 
+    // Create a unique uid base on the the given UID.
+    // If the given UID is not present in the file, the function return the given UID without any change.
+    // Otherwise, the function add a prefix to make the UID unique and return it
+    TIGL_EXPORT std::string MakeUIDUnique(const std::string& baseUID) const;
```

Comment:
Nice really helpful!

Maybe it would be nicer, if the number would be added as a suffix (e.g. for lexicographic sorting).